src/shared/ByteBuffer.h

```diff
@@ -436,9 +452,10 @@ class ByteBuffer
             append((uint8*)&value, sizeof(value));
         }
 
-    protected:
         size_t _rpos, _wpos;
         std::vector<uint8> _storage;
+
+        static constexpr size_t defaultSize = 0x1000;
```

Comment:
Code style question: I know we use ```m_someThing``` for members (at least in new code), and of course technically this is a member of the class as well. Now I personally find it rather handy to name statics differently so that you easily realize you're not just messing with that one object but potentially with all of them at the same time. Would people mind introducing something like, say, ```s_someThing``` for statics? Like we already use ```g_someThing``` in a few places for (typically) global stuff?

Reply:
I am down with s_